turtlebot_follower/launch/follower.launch

```diff
@@ -40,6 +40,43 @@
     </node>
   </group>
   <group if="$(arg simulation)">
-    <!-- not supported.. -->
+    <node pkg="nodelet" type="nodelet" name="follower_velocity_smoother"
+          args="standalone yocs_velocity_smoother/VelocitySmootherNodelet">
+      <rosparam file="$(find turtlebot_bringup)/param/defaults/smoother.yaml" command="load"/>
+      <remap from="follower_velocity_smoother/smooth_cmd_vel" to="cmd_vel_mux/input/navi"/>
+
+      <!-- Robot velocity feedbacks; use the default base configuration -->
+      <remap from="follower_velocity_smoother/odometry" to="/odom"/>
+      <remap from="follower_velocity_smoother/robot_cmd_vel" to="/mobile_base/commands/velocity"/>
+
+      <!-- connecting between follower/cmd_vel in mobile_base_nodelet and smoother/raw_cmd_vel in camera_nodelet -->
+      <!--     <remap from="follower_velocity_smoother/raw_cmd_vel" to="/mobile_base_nodelet_camera_nodelet/raw_cmd_vel"/> -->
+
+      <!-- Configure velocity smoother to allow more aggressive motion -->
+      <param name="accel_lim_v"  value="0.6"/>
+      <param name="accel_lim_w"  value="5.5"/>
+      <param name="decel_factor" value="2.0"/>
+    </node>
+
+    <param name="camera/rgb/image_color/compressed/jpeg_quality" value="22"/>
+
+    <!-- Make a slower camera feed available; only required if we use android client -->
+    <node pkg="topic_tools" type="throttle" name="camera_throttle"
+          args="messages camera/rgb/image_color/compressed 5"/>
+
+    <node pkg="nodelet" type="nodelet" name="turtlebot_follower"
```

Comment:
This block could be reused if instead of changing it to standalone, we simply launch a nodelet manager where it's expecting the manager at `/camera/camera_nodelet_manager` inside this branch of the if statement.